crypto/sigpb/sigpb.proto

```diff
@@ -44,10 +44,24 @@ message DigitallySigned {
     ECDSA = 3;
   }
 
+  // SignatureCipherSuite defines the set of algorithms used for signing.
+  enum SignatureCipherSuite {
```

Comment:
If the enums could deviate, declaring copies would be OK, but it doesn't seem we want that. I do think a top level enum would be more appropriate, considering it's used outside DigitallySigned, but I won't push any further.